homeassistant/components/notify/telegram.py

```diff
@@ -109,7 +109,8 @@ def send_message(self, message="", **kwargs):
         # send message
         try:
             self.bot.sendMessage(chat_id=self._chat_id,
-                                 text=title + "  " + message)
```

Comment:
You cannot discard the title because you don't use it. Please find another way to include it or do not remove this. 

We could disable the default title in the notify component but that will require some more changes